beluga/include/beluga/sensor/data/regular_grid.hpp

```diff
@@ -74,8 +74,12 @@ class BaseRegularGrid2 : public ciabatta::ciabatta_top<Derived> {
    * \return Grid cell coordinates.
    */
   [[nodiscard]] Eigen::Vector2i cell_near(double x, double y) const {
```

Comment:
@hidmic why is it important that this function is surjective? Is the intended co-domain all of the discrete plain or only the valid grid?

Reply:
It's not a requirement as much as a warning. Multiple $(x, y) \in \mathbb{R}^2$ tuples will map to the same $(x_i, y_i) \in \mathbb{Z}^2$ tuple.

> Is the intended co-domain all of the discrete plane or only the valid grid?

For a regular grid, the entire plane, as there is no notion of extent.

Reply:
With the new, closer to the original, replacement of floor() I don't need to change this comment.